[hotfix] Fix build (broken by change in PyTorch API)

bwasti opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* #48 [tvm] Use zero copy input
* **#47 [hotfix] Fix build (broken by change in PyTorch API)**

bwasti committed:
[hotfix] Fix build (broken by change in PyTorch API)

bwasti commented:
The fix for the PyTorch side is here (accepted and landing, so it should be safe to land this):
https://github.com/pytorch/pytorch/pull/21315

wanchaol reviewed (approved):
looks good, but ci failure seems to be build related?

bwasti committed:
Update on "[hotfix] Fix build (broken by change in PyTorch API)"

[hotfix] Fix build (broken by change in PyTorch API)

gh-metadata: pytorch tvm 47 gh/bwasti/31/head

bwasti committed:
Update on "[hotfix] Fix build (broken by change in PyTorch API)"

[hotfix] Fix build (broken by change in PyTorch API)

gh-metadata: pytorch tvm 47 gh/bwasti/31/head

bwasti committed:
Update on "[hotfix] Fix build (broken by change in PyTorch API)"

[hotfix] Fix build (broken by change in PyTorch API)

gh-metadata: pytorch tvm 47 gh/bwasti/31/head
